server/aws-lsp-codewhisperer/package.json

```diff
@@ -51,6 +51,7 @@
         "fast-glob": "^3.3.3",
         "fastest-levenshtein": "^1.0.16",
         "fdir": "^6.4.3",
+        "fs-extra": "^11.3.0",
```

Comment:
There really should be no need for fs-extra since I thought the [fs.ts](https://github.com/aws/aws-toolkit-vscode/blob/0819eb3bcfe591a7d18b71e9513856f27b75b10d/packages/core/src/shared/fs/fs.ts) module was ported from aws-toolkit-vscode, or was planned to be.

Adding fs-extra will cause problems with supporting web-mode, that's why we removed it from aws-toolkit-vscode. It's also not needed at all if you port fs.ts from aws-toolkit-vscode (which is more robust anyway).